compiler/dyno/lib/resolution/scope-queries.cpp

```diff
@@ -365,18 +366,21 @@ static bool doLookupInImports(Context* context,
         }
 
         // find it in that scope
-        bool found = doLookupInScope(context, symScope, nullptr, resolving,
-                                     from, newConfig,
-                                     checkedScopes, result);
-        if (found && onlyInnermost)
-          return true;
+        if (doLookupInScope(context, symScope, nullptr, resolving, from,
+                            newConfig, checkedScopes, result)) {
+          found = true;
+        }
```

Comment:
```suggestion
        found |= doLookupInScope(context, symScope, nullptr, resolving,
                                 from, newConfig, checkedScopes, result)
```
Might this be cleaner? I'm not sure which one I prefer, just a thought.

Reply:
That's actually what I had before my latest commit, but I looked into it and had some concern about performing bitwise OR to do a logical OR (and there is no ||= operator). From what I can tell ([shamelessly gathering my info from stackoverflow](https://stackoverflow.com/questions/24542/using-bitwise-operators-for-booleans-in-c)) it should work perfectly fine since bools can only be 0 or 1, and it passed all tests like that. I certainly prefer the compound assignment form for clarity, so maybe I'll go back to that.